### PR TITLE
Add publisher, producer, series, and author sort config options

### PR DESCRIPTION
**Pull request by anthonynelzin**
Related to #107, this PR adds the publisher and producer config options, to allow for a more flexible ePub output.

### CODE REVIEW

**Comment by azu** [comment]
It seems that snapshot testing is failed.
you need to update snapshot via `yarn run updateSnapshot`.
https://github.com/honkit/honkit/pull/108/checks?check_run_id=941363472

File: packages/honkit/src/__tests__/__fixtures__/honkit/docs/config.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ HonKit allows you to customize your book using a flexible configuration. These o
 | `title` | Title of your book, default value is extracted from the README. |
 | `description` | Description of your book, default value is extracted from the README. |
 | `author` | Name of the author. |
+| `producer` | Name of the producer. |
```
Comment (azu) [inline packages/honkit/src/__tests__/__fixtures__/honkit/docs/config.md:14]: Ah, This is fixture files.
You do not need to update this.

You can pass the tests via revert this change instead of update snapshots.
https://github.com/honkit/honkit/pull/108#issuecomment-668556185

Comment (anthonynelzin) [reply packages/honkit/src/__tests__/__fixtures__/honkit/docs/config.md:14]: OK, I'll do that!

**Comment by anthonynelzin** [comment]
That should be better, now.

**Comment by azu** [comment]
We have release [Release v3.6.0 · honkit/honkit](https://github.com/honkit/honkit/releases/tag/v3.6.0) 🎉
